go-controller/hybrid-overlay/pkg/controller/master_test.go

```diff
@@ -76,8 +74,6 @@ var _ = Describe("Hybrid SDN Master Operations", func() {
 		fexec = ovntest.NewFakeExec()
 		err := util.SetExec(fexec)
 		Expect(err).NotTo(HaveOccurred())
-
-		libovsdbCleanup = nil
```

Comment:
I think you want to keep this...

Reply:
We don't need to explicitly set that, it's generated when we create the client and is already declared  at the Test's global scope ->  https://github.com/ovn-org/ovn-kubernetes/blob/23f70762fce12455ee79f7485ec4c1d2d92cb14f/go-controller/hybrid-overlay/pkg/controller/master_test.go#L62

Reply:
@astoycos it's actually a global since we want to clean it up from AfterEach()... so we want to make sure to clear it every time, just in case some test *doesn't* call NewLibovsdb*TestHarness() and the pointer value at exit is from a previous test.

Reply:
Ah ok I see, that makes sense